docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+services:
+  quickstart:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        RIKAI_VERSION: 0.1.12
```

Comment:
the latest rikai release is 0.1.13

Reply:
yeah i made this PR prior to the release so the artifacts weren't ready. I'll update